test/integration/parachain/release/redeem.test.ts

```diff
@@ -86,56 +121,93 @@ describe("redeem", () => {
                 BITCOIN_CORE_PORT,
                 `vault_to_liquidate-${collateralCurrency.ticker}-${wrappedCurrency.ticker}`
             );
+
             // Steal some bitcoin (spend from the vault's account)
             const foreignBitcoinAddress = "bcrt1qefxeckts7tkgz7uach9dnwer4qz5nyehl4sjcc";
             const amountToSteal = newMonetaryAmount(0.00001, wrappedCurrency, true);
-            const btcTxId = await vaultBitcoinCoreClient.sendToAddress(foreignBitcoinAddress, amountToSteal);
-            // it takes about 15 mins for the theft to be reported
-            await DefaultTransactionAPI.waitForEvent(api, api.events.relay.VaultTheft, 17 * 60000);
+            const {txid: btcTxId} = await vaultBitcoinCoreClient.sendBtcTxAndMine(foreignBitcoinAddress, amountToSteal, 3);
+            // wait for tx inclusion and block finalization.
+            await waitForBlockFinalization(bitcoinCoreClient, userInterBtcAPI.btcRelay);
+
+            // report theft
+            await reporterInterBtcAPI.vaults.reportVaultTheft(vaultToLiquidateId, btcTxId);
+
+            const finalizedPromise = new Promise<void>((resolve, _) => userInterBtcAPI.system.subscribeToFinalizedBlockHeads(
+                async (header) => {
+                    const events = await userInterBtcAPI.api.query.system.events.at(header.parentHash);
+                    if (DefaultTransactionAPI.doesArrayContainEvent(events, api.events.relay.VaultTheft)) {
+                        resolve();
+                    }
+                })
+            );
+                
+            // wait for theft reported event
+            await finalizedPromise;
 
             const flaggedForTheft = await userInterBtcAPI.vaults.isVaultFlaggedForTheft(
-                vaultToLiquidateId.accountId,
-                currencyIdToLiteral(vaultToLiquidateId.currencies.collateral) as CollateralIdLiteral,
+                vaultToLiquidateId,
                 btcTxId
             );
-            assert.isTrue(flaggedForTheft);
+            assert.isTrue(
+                flaggedForTheft,
+                `Expected vault (collateral: ${collateralCurrency.ticker}) to be flagged for theft, but it was not.`
```

Comment:
Thank you. 
It would drive me bonkers if I saw a failure and wouldn't know which collateral variation failed without manually running the test with breakpoints. :)